### PR TITLE
MGMT-5443: Reduced reboot timeout to 40 minutes

### PR DESCRIPTION
**Pull request by eranco74**
# Assisted Pull Request

## Description

Since https://github.com/openshift/assisted-service/pull/2343 was merged reboot timeout leads to installing-pending-user-action status
So now that the timeout during reboot no longer fails the installation we can lower it to 40 minutes.

- Should this PR be tested by the reviewer? no
- Is this PR relying on CI for an e2e test run? yes for regression
- Should this PR be tested in a specific environment? no
- Any logs, screenshots, etc that can help with the review process? no

-->

## List all the issues related to this PR

- [x] New Feature
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [x] Automation (CI, tools, etc)
- [x] Cloud
- [x] Operator Managed Deployments
- [x] None

## How was this code tested?

- [x] assisted-test-infra environment - pending
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @avishayt 
/cc @tsorya 

## Checklist

- [x] Title and description added to both, commit and PR.
- [ x Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] Reviewers have been listed
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Review by flaper87 — APPROVED** [review]
:pizza:

**Comment by eranco74** [comment]
/hold
Need to test it with workers, 
Workers might hang longer in reboot state since they don't get their ignition until the control plane is ready.

**Comment by eranco74** [comment]
The worker reboot duration also relates to:
https://issues.redhat.com/browse/MGMT-7178

**Comment by eranco74** [comment]
Update, this timeout shouldn't affect workers, since they get updated to `Waiting for ignition` post-reboot.
However, reducing the timeout might trigger single node installation failures due to the time it takes the controller to start on a single node.
I'll update the PR to have a different reboot timeout for SNO

File: internal/host/hostutil/host_utils.go [hunk 1]
```diff
@@ -193,3 +195,22 @@ func UnmarshalConnectivityReport(reportStr string) (*models.ConnectivityReport,
...
+		log.WithError(err).Errorf("Failed to find cluster %s", host.ClusterID)
+		return nil, errors.Errorf("Failed to find cluster %s", host.ClusterID)
+	}
+	return &cluster, nil
+}
+
+func IsSingleNode(log logrus.FieldLogger, db *gorm.DB, host *models.Host) bool {
+	cluster, err := GetHostCluster(log, db, host)
+	if err != nil {
+		log.Debug("Can't check if host is part of single node OpenShift")
```
Comment (tsorya) [inline internal/host/hostutil/host_utils.go:212]: log.WithError?

Comment (eranco74) [reply internal/host/hostutil/host_utils.go:212]: The error is already logged in GetHostCluster

**Pull request merged**